docs/source/introduction.rst

```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
+    RESTful API to interact with it.  For more information, see
+    `<https://github.com/hyperledger/sawtooth-marketplace>`_.
+
+  * Sawtooth Private UTXO:
+    Experimental distributed application built on Hyperledger Sawtooth. This
+    example demonstrates how assets can be created and traded on the Sawtooth
+    distributed ledger. It shows how to use SGX to allow for assets to be
+    transferred off ledger and privately traded, where only the trading parties
+    know the details of the transaction. For more information, see
+    `<https://github.com/hyperledger/sawtooth-private-utxo>`_.
 
-Sawtooth abstracts the core concepts of consensus and isolates consensus
-from transaction semantics. The interface supports plugging in various
-consensus implementations. Sawtooth provides two such implementations:
-dev_mode and PoET.
 
-  * Dev_mode is a simplified random leader algorithm that is useful
-    for developers and test networks that require only crash fault tolerance.
+Getting Started with Application Development
+============================================
 
-  * PoET, short for “Proof of Elapsed Time" is a Nakamoto-style consensus
-    algorithm. It is designed to be a production-grade protocol capable of
-    supporting large network populations. Sawtooth includes an implementation
-    which simulates the secure instructions. This should make it easier for
-    the community to work with the software but also forgoes Byzantine fault
-    tolerance.  For more information, see
-    :doc:`PoET 1.0 Specification <../architecture/poet>`.
+Try Hyperledger Sawtooth
+------------------------
 
+The Sawtooth documentation explains how to set up a local validator for
+demonstrating Sawtooth functionality and for testing an application. Once
+running, you will be able to submit new transactions and fetch the resulting
+state and block data from the blockchain using HTTP and the Sawtooth REST API.
+These methods apply to the included example transaction families, as
+well as to any transaction families you might write yourself.
 
-Getting Sawtooth
-=====================
+Sawtooth validators can be run from pre-built Docker containers, installed
+natively using Ubuntu 16.04, or launched in AWS from the AWS Marketplace.
 
-The Sawtooth platform is distributed in source code form with
-an Apache license. You can get the code `here
-<https://github.com/hyperledger/sawtooth-core>`_ and start building your own
-distributed ledger.
+To get started, see :doc:`/app_developers_guide/installing_sawtooth`.
 
-Repositories
-============
+Develop a Custom Application
+----------------------------
 
-One repository contains all of the code needed:
+In Sawtooth, the data model and transaction language are implemented in a
+transaction family. Transaction families codify business rules used to modify
+state, while client programs typically submit transactions and view state. You
+can build custom transaction families that reflect the unique requirements of
```

Comment:
[ SHAWN ] Change to "custom transaction families that reflect your unique requirements."